### ISSUE TITLE
Brotli dependency retracted by author

### ISSUE THREAD

**Issue by st3fan** [gofiber/fiber#1352]
I'm not sure how important this is, but pulling in Fiber with `go get` results in a failure on Brotli. It looks like the the 1.0.1 version has been retracted.

```
$ go get github.com/gofiber/fiber/v2
```

```
go: downloading github.com/gofiber/fiber v1.14.6
go: downloading github.com/gofiber/fiber/v2 v2.10.0
go: downloading github.com/valyala/fasthttp v1.23.0
go: downloading golang.org/x/sys v0.0.0-20210225134936-a50acf3fe073
go: downloading github.com/valyala/tcplisten v0.0.0-20161114210144-ceec8f93295a
go: downloading github.com/andybalholm/brotli v1.0.1
go: downloading github.com/klauspost/compress v1.11.13
go: warning: github.com/andybalholm/brotli@v1.0.1: retracted by module author: occasional panics and data corruption
go: to switch to the latest unretracted version, run:
        go get github.com/andybalholm/brotli@latest
go get: added github.com/gofiber/fiber/v2 v2.10.0
```

**Comment by ReneWerner87** [comment]
Thx for the hint, we will check it

**Comment by hi019** [comment]
Will be fixed by #1351

**Comment by ReneWerner87** [comment]
please try https://github.com/gofiber/fiber/releases/tag/v2.11.0
